### DISCUSSION TITLE
Raspberry Pi 4 Shortages

### DISCUSSION THREAD

**Discussion by Dylan-Grove** [FundamentalFrequency/.github#23] [General]
With prices of PI 4 4GBs price going for $200 at MSRP of $35, I wonder if I could repurpose an old Pi 3B or 3B+

Do you think they would be powerful enough to run the LMN?

Side note: Any Canadians that want a PCB, let me know. I ordered the minimum of 5.

**Comment by stonepreston** [comment]
I have not tried it as I don't have a pi 3. So the answer is maybe, but can't really give you anything more than that.

**Reply by jshbrntt** [reply]
@stonepreston out of interest which model of the Raspberry Pi 4 did you use? The 4GB or 8GB model?

Also have you managed to top out the RAM with stacked plugins?

**Reply by Dylan-Grove** [reply]
> I have not tried it as I don't have a pi 3. So the answer is maybe, but can't really give you anything more than that.

I was able to install it on a 3 and run it with no issues. I'll update this thread once I'm able to build the whole board and do some thorough testing. Thanks for the reply.


> @stonepreston out of interest which model of the Raspberry Pi 4 did you use? The 4GB or 8GB model?

On the build guide:  

> The LMN-3-DAW has been tested on the 4 GB Raspberry Pi 4 only.

**Reply by jshbrntt** [reply]
Thanks for pointing out the model.

**Comment by jshbrntt** [comment]
I know this probably doesn't help most folk, but I live near Cambridge (UK) where Raspberry Pi's offices are located and they have an official store in the local mall.

However, be warned there is a strict store policy of **one per customer**.

Nice to see somebody combatting scalpers. Valve did a shoddy job with the Steam Deck.

I managed to snap up a 4GB model for £55 along with the Hyper 4.0 touchscreen also required for £51.

**Reply by deltronxzero** [reply]
I wonder how much shipping would cost to the states since thats a whole lot better than any of the prices of seen around!

**Reply by Dp666me** [reply]
> I wonder how much shipping would cost to the states since thats a whole lot better than any of the prices of seen around!

I shipped mine to Canada from there and it was 15 bucks

**Comment by nedhale** [comment]
@Dylan-Git I'd be interested in one of those ship-to-Canada PCBs if they're not all spoken for.

**Reply by Dylan-Grove** [reply]
Still have 4 left. Message me on discord and we can sort it out: PaullyBeenis#7481

**Reply by n0ughtchill** [reply]
Can you ship to US?

**Reply by Dylan-Grove** [reply]
> Can you ship to US?

If you want to pay for it sure, probably close to $20-30

**Comment by Dylan-Grove** [comment]
Update: I finally finished the build with my pi 3b+. It works totally fine so far. I need to do some more stress testing with lots of plug-ins and more tracks but I think you can get away with older models

**Reply by srwxr-xr-x** [reply]
Hey Dylan! 
I'm trying to run the LMN Daw on a Raspberry Pi 3b, but haven't been able to compile on it. I was also having trouble trying to cross compile, since I main windows and have never done it before. Would you mind sharing the binary that was built or at least sharing the steps that you used to compile it? I'm running Raspbian as well, didn't know if that was what you are using.
Thanks!

**Reply by srwxr-xr-x** [reply]
Nevermind, found the binary on github releases!

**Reply by Dylan-Grove** [reply]
Yeah that tripped me up as well. For anyone else experiencing issues running it, it's probably because you used the aarch version from the build guide. Use this one instead:
https://github.com/FundamentalFrequency/LMN-3-DAW/releases/download/v0.1.0/LMN-3-arm-linux-gnueabihf.zip

**Reply by iamdey** [reply]
Hi! Same here, I'm not able to find any Rpi 4 at decent price.

But I have a **Raspberry Pi 2b** and I was able to run LMN-3-DAW v0.1.0 in arm-linux arch (it doesn't support 64bit to my knowledge) and LMN-3-Emulator on a fresh raspbian installation.

I was able to make some basic LFO sounds but I'm still not comfortable with the emulator UI, I still need to go deeper in order to find a limit. 
Well that said, it's pretty encouraging that it might works «fine» on older systems (I am an enthousiastic slow tech person, reuse old stuff to make new one, because you know burning planet etc.) and I keep in mind that the build can be plugged to almost everything and hypothetically a [smartphone](https://github.com/orgs/FundamentalFrequency/discussions/42). 
So yes please keep the good work!

**Comment by parmour** [comment]
Hi,
it looks like my LMN-3 is working fine so far with a Ras Pi 3B.

I used the Bullseye OS, followed the build guide except using https://github.com/FundamentalFrequency/LMN-3-DAW/releases/download/v0.1.0/LMN-3-arm-linux-gnueabihf.zip , as suggested above.

The Hyperpixel4 worked without any extra drivers, but for a while I could not stop the display appearing in portrait mode, no matter what settings I put into  /boot/config.txt

Eventually adding:
`xrandr --output DSI-1 --rotate right`
to ~/LMN_start.sh seemed to resolve that!
